services/asset/nodes/models.go

```diff
@@ -20,13 +20,71 @@ type Node struct {
 	Nodes                   []Node                     `json:"nodes,omitempty"`
 	Type                    string                     `json:"type"`
 	Kind                    string                     `json:"kind,omitempty"`
+	Latitude                float64                    `json:"latitude,omitempty"`
 	NoradId                 string                     `json:"noradId,omitempty"`
-	InternationalDesignator string                     `json:"internationalDesignator,omitempty"`
 	Tle                     []string                   `json:"tle,omitempty"`
-	Latitude                *float64                   `json:"latitude,omitempty"`
-	Longitude               *float64                   `json:"longitude,omitempty"`
-	Elevation               *float64                   `json:"elevation,omitempty"`
+	InternationalDesignator string                     `json:"internationalDesignator,omitempty"`
+	Longitude               float64                    `json:"longitude,omitempty"`
+	Elevation               float64                    `json:"elevation,omitempty"`
 	NumberOfChildren        int                        `json:"numberOfChildren"`
+	PropertyList            []Property[any]            `json:"propertiesV2,omitempty"`
+}
+
+type Property[T any] struct {
```

Comment:
can't we reuse the one from asset/properties?

Reply:
It should be the same object that is shared, so:
1) we don't have to do changes twice
2) I believe it's the same in the implementation
But it's up to you here